lib/themes/dosomething/paraneue_dosomething/js/components/FeatureCard.js

```diff
@@ -23,6 +19,7 @@ class FeatureCard extends React.Component {
     this.campaigns = [];
     this.campaignsToExclude = setting('dsUser.activity.signups', []);
     this.campaignsApiPage = 1;
+    this.apiClient = new ApiClient(window.location.hostname, '/api/v1/', window.location.port);
```

Comment:
Do you ever see a scenario where we don't pass in these `window.location.hostname` & `window.location.port`? I feel like those could potentially be defaults